### ISSUE TITLE
Dialog: Ugly vertical scrollbar

### ISSUE THREAD

**Issue by Spencer857** [primefaces/primereact#7289]
### Describe the bug

In general, PrimeReact components are outstanding. Sadly, `Dialog` is an exception. Its vertical scrollbar looks like a bug, especially on smaller screens. Look at the Bootstrap equivalent called [modal](https://getbootstrap.com/docs/5.3/components/modal/#scrolling-long-content). Their dialog doesn't have a dedicated scrollbar. To scroll, you use the window scrollbar.

### System Information

```Shell
primereact: 10.8.3
```

**Comment by melloware** [comment]
Can you post a screenshot of what you mean?

**Comment by melloware** [comment]
Oh I see you don't want an inner scrollbar on the Dialog itself.  I am pretty sure all 4 PrimeTek libs do this so I am not sure they will make any changes

**Comment by sja-cslab** [comment]
For me its perfect the way it is. The way bootstrap handles that look weird to me.
The content of the Dialog is long so scroll the dialog. not the page. The difference here is, that the bootstrap modal/dialog goes outside of main page.

However if you look onto the second long content example - you'll notice that they've the `.modal-dialog-scrollable` class which causes the inner scrollbar like PrimeReact does.

**Comment by Spencer857** [comment]
Depending on the use case, one behaves better than the other. I think PrimeReact should provide an additional boolean property to allow a choice.

**Comment by sja-cslab** [comment]
I get your point however - dialogs larger than screen feels like anti-pattern

**Comment by melloware** [comment]
Yeah I personally think PrimeReact handles it correctly and the Bootstrap using the browser scrollbar is wrong but that is just my vote and my two cents.
